pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/security/ApexCRUDViolationRule.java

```diff
@@ -191,42 +189,41 @@ public Object visit(ASTUserClass node, Object data) {
         return super.visit(node, data);
     }
 
-    @Override
-    public Object visit(ASTMethodCallExpression node, Object data) {
+    @Override public Object visit(ASTMethodCallExpression node, Object data) {
         if (Helper.isAnyDatabaseMethodCall(node)) {
 
             if (hasAccessLevelArgument(node)) {
                 return data;
             }
 
             switch (node.getMethodName().toLowerCase(Locale.ROOT)) {
-            case "insert":
```

Comment:
this looks quite off, being on the same level, despite having checkstyle, right?

@adangel
@zbynek

Reply:
My contribution to PMD is minimal (1 accepted PR and a few  comments), so I'm definitely not the right person to decide the code style for PMD.

That being said, this is aligned with Sun's code style: https://checkstyle.sourceforge.io/styleguides/sun-code-conventions-19990420/CodeConventions.doc6.html#a468
and for projects that use 4 space indentation it IMHO makes sense. On the other hand, Google code style indents the cases: https://google.github.io/styleguide/javaguide.html#s4.8.4-switch (but it also uses 2 space indentation).

Reply:
> Sun's code style

crazy to see, thanks for that.

Yes of course we not going to decide, we only can suggest wether auto fix is a nice thing to have to rather we want to fix these trivial issues by hand like its currently approached. That beging sad the current setup feels like 2015 but after a decade of fixing the silliness myself, finally theres an app for that imposing convention of configuration.

Is the convention of configuration principle something you would appreciate in order to improve the dev exp.

- https://github.com/pmd/pmd/discussions/5875

Reply:
> was this intended

of course not.

If the cost of all config burden is vanished for the price to waive some beloved spaces before array init, as example, the tradeoff would be quite simple, as least for me.

Its a lot to talk about this and then. When the answer is always we use some others convention the discussion seems boringly done.

So its up to you do balance the price for the caseIndent over the cost for the current configuration imposed.

A simple convention, ignoring its randomly details, makes the best config obsolete.